src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-04-17-studio-73-11-1.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'April 17, 2018'
+description: .
+tree_item_index: 988
+review:
+  comment: ''
+  date: '2018-04-17'
+  status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v73-11-1'}}
+This release eases the document type creation process when using Web UI.
+{{! /multiexcerpt}}
+
+## Easier Document Type Creation Process
+When configuring a document type, a new `SVG icon` option is now available if your application has a dependency to the `Web UI` addon.
```

Comment:
When configuring a document type, the `SVG icon` option is now available if your application has a dependency to the `Web UI` addon.